data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/util/AuthenticationPropertyConfigurer.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.kafka.util;
+
+import org.opensearch.dataprepper.plugins.kafka.configuration.KafkaSinkConfig;
+
+import java.util.Base64;
+import java.util.Properties;
+
+/**
+ * * This is static property configurer dedicated to authencation related information given in pipeline.yml
+ */
+
+public class AuthenticationPropertyConfigurer {
```

Comment:
Looks like this is OAUTH specific. So, the class should be named appropriately.

Reply:
hi @kkondaka  this class includes authentication properties for sasl_plaintext as well so that is why we have used such name.